packages/guideline-themes-settings/package.json

```
@@ -0,0 +1,86 @@
+{
+    "name": "@frontify/guideline-themes-settings",
```

Isn't what Mike proposed? so we merge `app-bridge-theme` and `guideline-themes` into a single package? 
```suggestion
    "name": "@frontify/theme-brand-sdk",
```

---

Sure, let's align with how platform app is then (without `-settings`), blocks team has also in their backlog the task to remove `-settings` to just keep `@frontify/guideline-blocks`.

```suggestion
    "name": "@frontify/guideline-themes",
```
(+ folder rename)